src/components/community/DesktopLeftNav.jsx

```
@@ -126,27 +106,43 @@ export default function DesktopLeftNav() {
+                      <ContextMenuTrigger asChild>
+                        <a
+                          href={`/community?tab=brand&brandId=${encodeURIComponent(b.brandId)}&brand=${encodeURIComponent(label)}&via=left_rail`}
+                          onClick={(e) => {
+                            e.preventDefault();
+                            try { track('community_leftrail_click', { brandId: b.brandId }); } catch (err) { console.debug?.('DesktopLeftNav track leftrail click failed', err); }
+                            const next = new URLSearchParams(location.search);
+                            next.set('tab', 'brand');
+                            next.set('brandId', b.brandId);
+                            next.set('brand', label);
+                            navigate({ pathname: location.pathname, search: next.toString() });
+                            try { markAsRead?.(b.brandId); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)